kafka-service/pom.xml

```diff
@@ -5,12 +5,12 @@
     <parent>
         <artifactId>dispatcher-suite</artifactId>
         <groupId>edu.cornell.eipm.messaging.microservices</groupId>
-        <version>1.4.1-SNAPSHOT</version>
+        <version>1.4.2-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>kafka-service</artifactId>
-    <version>1.4.7</version>
+    <version>1.4.6</version>
```

Comment:
Shouldn't this be `1.4.8`?

Reply:
Since when versions are advanced forward?  :)

Great catch, anyway . Fixed here: https://github.com/eipm/DispatcherSuite/pull/37/commits/3283e60357a8ad4d74a9ae531d7668b9882d71f6